tests/platforms/kubernetes/app_description.go

```diff
@@ -62,6 +69,11 @@ func (a AppDescription) String() string {
 	return string(j)
 }
 
+// ShouldBeExposed returns if the app should be exposed as a loadbalancer/nodeport service.
+func (a AppDescription) ShouldBeExposed() bool {
+	return a.IngressEnabled && !strings.EqualFold(os.Getenv(useServiceInternalIP), "true")
```

Comment:
Nit: rather than checking for value `true`, please use `utils.IsTruthy` which checks for other allowed values too (like `1`, `y`, `yes`...)

Reply:
fixed!